.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
clang is a cross compiler at it's core. This means there aren't different binaries for cross compilation but it is specified as a -target argument. The Waf build system has the toolchain that should be used for the different targets but it is generating the name for the binary to be used. Should Waf be changed to support clang or should symbolic links be created instead?